compiler/model/utils.ts

```diff
@@ -547,6 +547,31 @@ export function hoistRequestAnnotations (
     } else if (tag === 'since') {
       assert(jsDocs, semver.valid(value), `Request ${request.name.name}'s @since is not valid semver: ${value}`)
       endpoint.since = value
+    } else if (tag === 'security_prerequisites_index') {
+      const privileges = [
```

Comment:
can we reuse the values from this: https://github.com/elastic/elasticsearch-specification/pull/718 -- seems like an anti pattern to maintain the same values in two places, same for cluster below

Reply:
Unfortunately not, as the `specification` and the `compiler` are two separate typescript projects, one cannot import the other, as the `tsconfig.json` is very different.

Reply:
that's unfortunate!